modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/resource/releases/SW360Release.java

```diff
@@ -252,12 +254,12 @@ public SW360Release setCopyrights(String copyrights) {
         return this;
     }
 
-    public SW360Release setExternalIds(Map<String,String> externalIds) {
+    public SW360Release setExternalIds(Map<String, String> externalIds) {
```

Comment:
I think the old code looks better? Is this the auto-formatter?

Reply:
I think so, although a space after comma is standard, isn't it?